CODEOWNERS

```diff
@@ -1 +0,0 @@
-@jmazanec15 @vamshin @VijayanB @svencowart @VachaShah
```

Comment:
Actual codeowners file already exists in `.github` folder. This might have been added by mistake.